content/bn/docs/reference/glossary/kube-apiserver.md

```diff
@@ -0,0 +1,23 @@
+---
+title: API সার্ভার
+id: kube-apiserver
+date: 2018-04-12
+full_link: /bn/docs/concepts/overview/components/#kube-apiserver
+short_description: >
+   কন্ট্রোল প্লেন উপাদান যা কুবারনেটিস API পরিবেশন করে।
+
+aka:
+- kube-apiserver
+tags:
+- architecture
+- fundamental
+---
+ API সার্ভার হলো কুবারনেটিস {{< glossary_tooltip text="কন্ট্রোল প্লেন" term_id="control-plane" >}}
+এর একটি উপাদান যা কুবারনেটিস API কে প্রকাশ করে ৷
+API সার্ভার কুবারনেটিস কন্ট্রোল প্লেনের ফ্রন্ট এন্ড হিসেবে কাজ করে।
+
+<!--more-->
+
+কুবারনেটিস API সার্ভারের প্রধান বাস্তবায়ন হলো [kube-apiserver](/bn/docs/reference/generated/kube-apiserver/)।
```

Comment:
কুবারনেটিস API সার্ভারের প্রধান বাস্তবায়ন হলো [kube-apiserver](/bn/docs/reference/generated/kube-apiserver/🤚।

replace 🤚 with )

Reply:
@MRH-Romit  Sorry for the inconvenience .  Please fix this .